rotkehlchen/history/events.py

```diff
@@ -577,7 +587,13 @@ def fail_history_cb(error_msg: str) -> None:
 
         # include the ledger actions
         self.processing_state_name = 'Querying ledger actions history'
-        ledger_actions, _ = self.query_ledger_actions(has_premium, from_ts=None, to_ts=end_ts)
+        # Query ledger actions taking into account premium limits.
+        ledger_actions, _ = self.query_ledger_actions(
+            has_premium,
+            from_ts=None,
+            to_ts=end_ts,
+            new_exchange_ledger_actions=ledger_actions,
```

Comment:
Where are these `ledger_actions` defined? Inside the if here: https://github.com/rotki/rotki/pull/3481/files#diff-dcdada8d937a3b75365eebb25d35359cbafbc6e3c5f0b21e5c6baf3a342f8cc5R518
? So they can end up being undefined?

Reply:
The default empty list value is at L501